site/content/en/docs/Guides/feature-stages.md

```diff
@@ -42,6 +43,25 @@ The current set of `alpha` and `beta` feature gates:
 | [Extend Webhook autoscaler to send fleet metadata with the request](https://github.com/googleforgames/agones/issues/3951)  | `FleetAutoscaleRequestMetaData`    | Disabled | `Alpha`  | 1.48.0 |
 | [Sidecar Containers](https://github.com/googleforgames/agones/issues/3642)                                                 | `SidecarContainers`                | Disabled | `Alpha`  | 1.49.0 |
 | Example Gate (not in use)                                                                                                  | `Example`                          | Disabled | None     | 0.13.0 |
+{{% /feature %}}
+
+{{% feature publishVersion="1.51.0" %}}
+| Feature Name                                                                                                               | Gate                               | Default  | Stage    | Since  |
+|----------------------------------------------------------------------------------------------------------------------------|------------------------------------|----------|----------|--------|
+| [CountsAndLists](https://github.com/googleforgames/agones/issues/2716)                                                     | `CountsAndLists`                   | Enabled  | `Beta`   | 1.41.0 |
+| [DisableResyncOnSDKServer](https://github.com/googleforgames/agones/issues/3377)                                           | `DisableResyncOnSDKServer`         | Enabled  | `Stable` | 1.49.0 |
+| [Support Passthrough on GKE Autopilot](https://github.com/googleforgames/agones/issues/3721)                               | `AutopilotPassthroughPort`         | Enabled  | `Beta`   | 1.43.0 |
+| [Support for Extended Duration Pods on GKE Autopilot (*1.28+ only*)](https://github.com/googleforgames/agones/issues/3386) | `GKEAutopilotExtendedDurationPods` | Enabled  | `Beta`   | 1.44.0 |
+| [Port Policy None](https://github.com/googleforgames/agones/issues/3804)                                                   | `PortPolicyNone`                   | Enabled  | `Beta`   | 1.49.0 |
+| [Multiple dynamic port ranges](https://github.com/googleforgames/agones/issues/1911)                                       | `PortRanges`                       | Enabled  | `Beta`   | 1.49.0 |
+| [Rolling Update Fixes](https://github.com/googleforgames/agones/issues/3688)                                               | `RollingUpdateFix`                 | Disabled | `Beta`   | 1.50.0 |
```

Comment:
Shouldn't this feature gate `RollingUpdateFix` defaulted to Enabled as it's in Beta too ? @0xaravindh